Gruntfile.js

```diff
@@ -278,16 +279,16 @@ var _              = require('lodash'),
                                 return 'echo Installing client dependencies... && npm install';
 
                             case 'prod':
-                                return './node_modules/.bin/ember build --environment=production --silent';
+                                return emberPath + ' build --environment=production --silent';
 
                             case 'dev':
-                                return './node_modules/.bin/ember build --silent';
+                                return emberPath + ' build --silent';
 
                             case 'test':
-                                return './node_modules/.bin/ember test --silent';
+                                return emberPath + ' test --silent';
 
                             default:
-                                return './node_modules/.bin/ember build --silent';
+                                return emberPath + ' build --silent';
```

Comment:
Hey actually, want to just delete this default stanza? it's dead code that isn't used anywhere